packages/sdk/src/types/Customers.ts

```diff
@@ -136,3 +137,56 @@ export type CustomersUpdateParams = CustomerRequest & IdOrSourceId
 export type CustomersUpdateResponse = CustomerObject | CustomerUnconfirmed
 
 export type CustomersUpdateConsentsBody = Record<string, boolean>
+
+// New types following guideline
+
+// Domain types
+
+type CustomerBase = {
+	name?: string | null
+	description?: string | null
+	email?: string | null
+	phone?: string | null
+	birthdate?: string | null
+	metadata?: Record<string, any> | null
```

Comment:
Change "any" to "unknown" -> you still can assign here anything, but it does not turn off all TS safeguards when you read it